utils/nctl/sh/scenarios/common/itst.sh

```diff
@@ -296,7 +296,12 @@ function get_switch_block() {
     fi
 
     if [ "$WALKBACK" -gt 0 ]; then
-        BLOCK=$(echo "$JSON_OUT" | jq '.result.block')
+        VERSIONED_BLOCK="$(echo "$JSON_OUT" | jq '.result.block_with_signatures.block')"
+        if jq -e '.Version2' <<< "$VERSIONED_BLOCK" >/dev/null; then
+            BLOCK="$(echo "$VERSIONED_BLOCK" | jq '.Version2')"
+        else
+            BLOCK="$(echo "$VERSIONED_BLOCK" | jq '.Version1')"
+        fi
```

Comment:
might be worth pulling out to a function, we repeat this a lot now

Reply:
yes, that's potentially worth doing, but I'm also not pushing too much because we'll need to rework this stuff in the future when we'll be working on test coverage for all legacy data types